workloads/free5gc/free5gc-cp/ausf/ausf-configmap.yaml

```diff
@@ -7,34 +6,4 @@ metadata:
     app.kubernetes.io/version: "v3.1.1"
     app: free5gc
 data:
-  ausfcfg.yaml: |
-    info:
-      version: 1.0.2
-      description: AUSF initial local configuration
-
-    configuration:
-      serviceNameList:
-        - nausf-auth
-      
-      sbi:
-        scheme: http
-        registerIPv4: ausf-nausf  # IP used to register to NRF
-        bindingIPv4: 0.0.0.0      # IP used to bind the service
-        port: 80
-        tls:
-          key: config/TLS/ausf.key
-          pem: config/TLS/ausf.pem
-      
-      nrfUri: http://nrf-nnrf:8000
-      plmnSupportList:
-        - mcc: 208
-          mnc: 93
-        - mcc: 123
-          mnc: 45
-      groupId: ausfGroup001
-      eapAkaSupiImsiPrefix: false
-
-    logger:
-      AUSF:
-        ReportCaller: false
-        debugLevel: info
+  ausfcfg.yaml: "info:\n  version: 1.0.2\n  description: AUSF initial local configuration\n\nconfiguration:\n  serviceNameList:\n    - nausf-auth\n  \n  sbi:\n    scheme: http\n    registerIPv4: ausf-nausf  # IP used to register to NRF\n    bindingIPv4: 0.0.0.0      # IP used to bind the service\n    port: 80\n    tls:\n      key: config/TLS/ausf.key\n      pem: config/TLS/ausf.pem\n  \n  nrfUri: http://nrf-nnrf:8000\n  plmnSupportList:\n    - mcc: 208\n      mnc: 93\n    - mcc: 123\n      mnc: 45\n  groupId: ausfGroup001\n  eapAkaSupiImsiPrefix: false\n\nlogger:\n  AUSF:\n    ReportCaller: false\n    debugLevel: info\n"
```

Comment:
I guess this is what yamlfmt does but it ruins the readability here. Any way to avoid this formatting?